integration/helpers/create-fixture.ts

```diff
@@ -287,11 +287,6 @@ export async function createFixtureProject(
 
   await fse.ensureDir(projectDir);
   await fse.copy(integrationTemplateDir, projectDir);
-  await fse.copy(
-    path.join(__dirname, "../../build/node_modules"),
-    path.join(projectDir, "node_modules"),
-    { overwrite: true }
-  );
```

Comment:
This throws an error when combined with pnpm's use of symlinks. As an alternative approach, we now always copy `build` to `dist` and then use `workspace:*` dependencies so that the built versions of packages are available within workspace packages.